thavalon-webapp/src/components/gameComponents/playerBoard.tsx

```diff
@@ -80,21 +99,50 @@ export function PlayerBoard(): JSX.Element {
      * @param message The GameMessage from the server
      */
     function handleGameMessage(message: GameMessage): void {
+        setGamePhase(mapMessageToGamePhase(message.messageType));
```

Comment:
We should probably also report `Error` messages, since that's how the backend tells the player they did something wrong (like propose the wrong number of players)